inscriptions/ISic004283.xml

```diff
@@ -76,27 +80,30 @@
                     <physDesc>
                         <objectDesc>
                             <supportDesc>
-                                <support>
-                                    <dimensions><!--Default values and requires editing-->
-                                        <height unit="cm"/>
-                                        <width unit="cm"/>
-                                        <depth unit="cm"/>
+                                <support><p>Stele of volcanic stone</p>
```

Comment:
`<p>Plaque of volcanic stone, re-used at least once. This is the same support as <ref sameAs="ISic003988.xml">ISic003988 (I.Lipara 373)</ref></p>`